tests/PropertySet_2.py

```diff
@@ -357,10 +357,12 @@ def testToDict(self):
 
         self.assertIsInstance(d2["bool"], bool)
         self.assertEqual(d2["bool"], True)
+        # Newer versions of SWIG can return int here
```

Comment:
I think there is a later commit to remove this line now that SWIG has been updated and we can't go back. @ktlim asked for a note in the code there so we would understand why the test fails. In modern SWIG they come back as int. He did not want a generic test for `number.Integral`.